vue-frontend/src/components/ios-app-development/SuccessStory.vue

```diff
@@ -0,0 +1,255 @@
+        :srcset="`${reviewBackgroundImage} 400w`"
+        class="tw-h-full tw-w-full tw-object-cover tw-ease-in-out tw-duration-1000"
+        alt="ios-review-image"
+      />
+      <div
+        class="tw-container swiper-content tw-absolute tw-p-4 tw-w-full tw-text-center tw-text-[#FFF] tw-transform tw-left-1/2 tw-top-1/2 tw--translate-x-1/2 tw--translate-y-1/2 xl:tw-w-[80%] md:tw-top-[49%] lg:tw-top-[48%]"
+      >
+        <swiper
+          :slidesPerView="1"
+          :centeredSlides="true"
+          :autoplay="{
+            delay: 3000,
+            disableOnInteraction: false,
+          }"
+          :speed="3000"
+          :loop="true"
+          :spaceBetween="20"
+          :pagination="pagination"
+          :modules="modules"
+          class="swiper-container"
+        >
+          <swiper-slide
+            v-for="(client, index) in clients"
+            :key="index"
+            class="tw-cursor-pointer tw-my-auto"
+          >
+            <div
+              class="tw-mx-auto sm:tw-px-10 md:tw-px-6 lg:tw-px-4 xl:tw-px-24 tw-text-white tw-flex tw-flex-col tw-justify-between tw-items-center tw-object-center"
+            >
+              <div
+                class="tw-font-inter-semibold tw-text-lg tw-leading-[1.6875rem] tw-transition-all tw-ease tw-duration-500 tw-tracking-[-2%]"
+              >
+                {{ client.review }}
+              </div>
+              <div class="tw-flex tw-flex-col tw-mt-8 tw-mb-2.5">
+                <span class="tw-flex tw-flex-row">
+                  <font-awesome-icon
+                    v-for="i in 5"
+                    :key="i"
+                    class="fa-star tw-w-5 tw-h-5 footer-icon"
+                    icon="star"
+                /></span>
+              </div>
+              <div class="tw-flex tw-flex-col">
+                <span
+                  class="tw-font-inter-semibold tw-text-base tw-leading-[1.2rem]"
+                  >{{ client.name }}</span
+                >
+                <span
+                  class="tw-font-inter-regular tw-text-xs tw-leading-[0.9rem] tw-mt-[0.3rem]"
+                  >{{ client.designation }}</span
+                >
+              </div>
+            </div>
+          </swiper-slide>
+        </swiper>
+      </div>
+    </div>
+    <!-- Mobile UI END -->
+    <!-- Desktop UI -->
+    <div
+      class="tw-hidden tw-w-full tw-h-[850px] lg:tw-h-[750px] tw-bg-[#FFF8F5] md:tw-flex tw-justify-center tw-items-center tw-relative tw-overflow-hidden before:tw-content-[''] before:tw-absolute before:tw-bottom-[0%] lg:before:tw-bottom-[-8%] before:tw-w-full before:tw-bg-[#fff] before:tw-h-[7%] before:tw-rounded-[50%] before:tw-shadow-[15px_1px_0_100px_rgba(255,255,255)] after:tw-content-[''] after:tw-absolute after:tw-top-[-2%] lg:after:tw-top-[-11%] after:tw-w-full after:tw-bg-[#fff] after:tw-h-[10%] after:tw-rounded-[50%] after:tw-shadow-[15px_1px_0_100px_rgba(255,255,255)] top-curve xll:tw-container"
+    >
+      <swiper
```

Comment:
please make sure this whole content to be in center

Reply:
Yes, its in center only.

Reply:
no it is not see this image the space before current card and after current text is not same

Reply:
because this card is rotated. that's why it looks like this